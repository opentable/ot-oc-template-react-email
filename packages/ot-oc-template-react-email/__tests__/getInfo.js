const { getInfo } = require("../index");

test("should return the correct info", () => {
  const info = getInfo();
  delete info.version;
  expect(info).toMatchSnapshot();
});
