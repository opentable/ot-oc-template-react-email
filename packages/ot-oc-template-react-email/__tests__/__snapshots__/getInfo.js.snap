// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return the correct info 1`] = `
Object {
  "externals": Array [
    Object {
      "global": "PropTypes",
      "name": "prop-types",
      "url": "https://unpkg.com/prop-types@15.6.1/prop-types.min.js",
    },
    Object {
      "global": "React",
      "name": "react",
      "url": "https://unpkg.com/react@16.2.0/umd/react.production.min.js",
    },
    Object {
      "global": "ReactDOM",
      "name": "react-dom",
      "url": "https://unpkg.com/react-dom@16.2.0/umd/react-dom.production.min.js",
    },
  ],
  "type": "ot-oc-template-react-email",
}
`;
